Getting Started.md

```diff
@@ -839,6 +885,116 @@ You should now be able to build and run this perk to see it at work.
 
 This should be a proper introduction into effects and their powers. Feel free to discuss with us on the our modding Discord if you need any assistance.
 
+# Publishing The Mod
+
+Once you are happy with your Mod, it is time to publish it to the world. This example will show you how to publish your Mod to the [Thunderstore ATS Mod Page](https://thunderstore.io/c/against-the-storm/).
```

Comment:
For consistency, use `mod` in lowercase. Do so in all instances.

Reply:
Adjusted.